### DISCUSSION TITLE
Uniformize lemma names in the standard library

### DISCUSSION THREAD

**Discussion by utaal** [verus-lang/verus#742] [General]
Current plan is:

* [x] `lemma_*` prefix
* [x] make lemmas associated functions when possible

[from the Verus meeting]

**Comment by parno** [comment]
I think we've largely done this?  If so, should we keep this open as documentation or close it?

**Reply by utaal** [reply]
Closing is good!

Perhaps we can add a note in `CONTRIBUTING.md` about the convention?

**Comment by parno** [comment]
Done!  ([0684141](https://github.com/verus-lang/verus/commit/0684141eb759bef1b00763b732d0f1f63978204c))

**Reply by utaal** [reply]
Thank you!
